draft-ietf-oauth-identity-chaining.md

```diff
@@ -65,26 +65,30 @@ This specification defines a mechanism to preserve identity and authorization in
 
-## Requirements Language
+# Conventions and Definitions
 
 {::boilerplate bcp14-tagged}
 
+This document uses the following terms:
+
+* Actor
+
+The role of "actor" represents any component in the OAuth system that performs the steps outlined in this document. While this document uses the term "actor" for clarity and to distinguish different roles and clients, in other contexts it should be referred to as "Actor in scope of Identity and Authorization Chaining".
+
 # Identity and Authorization Chaining Across Domains
 
 This specification describes a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JWT Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}} to achieve identity and authorization chaining across domains.
 
-A client in trust domain A that needs to access a resource server in trust domain B requests a JWT authorization grant from the authorization server for trust domain A via a token exchange. The client in trust domain A presents the received grant as an assertion to the authorization server in domain B in order to obtain an access token for the protected resource in domain B. The client in domain A may be a resource server, or it may be the authorization server itself.
-
 ## Overview
 
-The identity and authorization chaining flow outlined below describes how a combination of OAuth 2.0 Token Exchange {{RFC8693}} and JWT Profile for OAuth 2.0 Client Authentication and Authorization Grants {{RFC7523}} are used to address the use cases identified. The appendix include two additional examples that describe how this flow is used. In one example, the resource server acts as the client and in the other, the authorization server acts as the client.
+The identity and authorization chaining flow outlined below describes the steps the actor in Domain A performs to retrieve an access token for Domain B. The appendix include two concrete examples that describe how this flow is used. In one example, the resource server is the actor and in the other, the authorization server takes that role.
```

Comment:
I'm not sure the last sentence is correct. The software component, or Actor, can perform both the `resource server` OAuth role and the `client` OAuth role.